notebooks/standardize_events_data.ipynb

```diff
@@ -690,22 +1309,22 @@
     "                'timestamp': data_list[15]\n",
```

Comment:
There's a lot of variables that we actually don't need. Check src.hashstack_v0.py to see what's needed and what's not needed. Ignore variables which are not needed. Holds for Hashstack V1, too.